pages/events/index.vue

```diff
@@ -7,6 +7,7 @@ import _get from 'lodash/get'
 
 // GQL
 import FTVAEventList from '../gql/queries/FTVAEventList.gql'
+import { useIndexAggregator } from '~/composables/useIndexAggregator'
```

Comment:
This import is not required as anything composables directory is auto imported by nuxt